### PR TITLE
add pep pointer tool

### PR DESCRIPTION
**Pull request by caleb-easterly**
This is @pravs3683's PepPointer tool to classify novel peptides based on their genomic coordinates.

### CODE REVIEW

File: tools/pep_pointer/pep_pointer.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<tool id="pep_pointer" name="PepPointer: classify genomic location of peptides" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="2.7.9">python</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+        python $__tool_directory__/pep_pointer.py "$gtf" "$bed" "$classified"
```
Comment (bgruening) [inline tools/pep_pointer/pep_pointer.xml]: single quotes, and `$__tool_directory__/pep_pointer.py` should also be single quoted

Comment (caleb-easterly) [reply tools/pep_pointer/pep_pointer.xml]: what's the difference between single quotes and double quotes in...Cheetah? I'm used to shell, where `echo '$gtf'` gives `$gtf`

Comment (caleb-easterly) [reply tools/pep_pointer/pep_pointer.xml]: (I'm just the wrapper, so I'm not sure about BEDtools)

Comment (bgruening) [reply tools/pep_pointer/pep_pointer.xml]: Its more a bash'ism. Single quotes are not escaped/evaluated in bash, double quotes are. So double quotes do have a potential security impact if a user has ENV vars in there text params or filenames etc ...

Comment (bgruening) [reply tools/pep_pointer/pep_pointer.xml]: ping @jj-umn for the BEDtools Q.

**Comment by caleb-easterly** [comment]
We're using PepPointer in a publication, so it would be useful to have it released (even if we determine later on that it duplicates some BEDtools functionality)

**Pull request merged**